test/e2e.sh

```diff
@@ -0,0 +1,95 @@
+#! /bin/bash
+
+# Get running RHCOS version
+find_rhcos_version() {
+    worker_node=$(kubectl get nodes --selector='node-role.kubernetes.io/worker' -ojsonpath={.items[].metadata.name})
+    oc debug node/${worker_node} -- cat /host/etc/os-release  > ${tmpdir}/etc-os-version
```

Comment:
This may be easier to get the RHCOS version:

> oc get nodes/dgxa100 -ojsonpath={.status.nodeInfo.osImage} | awk '{ print $6 }'

except if there's a good reason to jump into the host?

Reply:
> Or is it better to avoid the overhead and complexity of using a debug pod here

the complexity is rather limited, to be honest :) 

> Wanted to get the RHCOS version from the exact same spot that NFD uses to create the label

this argument is convincing for sure, let's keep it this way then

> `> ${tmpdir}/etc-os-version`

I could remove this `tmpdir` and use variable instead, but nothing strong about it

Reply:
Thanks, I left it as an oc debug. 

I rewrote the function to get rid of the global rhcos_version variable and the tmpdir and file.